corehq/apps/app_manager/tests/test_media_suite.py

```diff
@@ -131,7 +131,6 @@ def test_form_media_with_app_profile(self, *args):
             app.create_mapping(CommCareImage(_id='form_image_{}'.format(i)), path, save=False)
 
         app.set_media_versions()
-        app.update_media_language_map()
         app.remove_unused_mappings()
```

Comment:
Is this needed now? `remove_unused_mappings`?

Reply:
The test seems to pass either way, but I'd rather not change it as part of this PR.

Reply:
i thought `remove_unused_mappings` was used to update multimedia_map and that was removed in this PR so i assumed this was related. I think i mixed `multimedia_map` and `multimedia_language_map`.
nvm.

Reply:
I've been confusing those two pretty often myself...